KubeArmor/build/kubearmor-test-k3s.yaml

```diff
@@ -0,0 +1,1065 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubearmor
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubearmor
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: kubearmor
+  namespace: kube-system
+---
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubearmor
+  namespace: kube-system
+  labels:
+    kubearmor-app: kubearmor
+spec:
+  selector:
+    matchLabels:
+      kubearmor-app: kubearmor
+  template:
+    metadata:
+      labels:
+        kubearmor-app: kubearmor
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/kubearmor: unconfined
+    spec:
+      serviceAccountName: kubearmor
+      nodeSelector:
+        kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      hostPID: true
+      hostNetwork: true
+      restartPolicy: Always
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: kubearmor
+        image: kubearmor/kubearmor:latest
+        imagePullPolicy: Always
```

Comment:
If we use "Always", we cannot test a local image
since the latest image in the Docker Hub will be used instead of a local one.
We need to use "Never" here.

Reply:
Ah, right! I will make the change. Thanks for pointing.